### ISSUE TITLE
Example code downloading lephare data gives: TypeError: get_auxiliary_data() got an unexpected keyword argument 'clone'

### ISSUE THREAD

**Issue by hdante** [lephare-photoz/lephare#179]
Hello, when running the example code:

```python
import os
os.environ['LEPHAREDIR']='/path/to/my/preferred/directory/'
# You must import lephare after setting the variables
import lephare as lp
# If you do not set a config input to the following function in gets everything.
lp.data_retrieval.get_auxiliary_data(clone=False)
```

Gives:

```python3
Python 3.11.9 (main, Apr 19 2024, 16:48:06) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ['LEPHAREDIR']='/lustre/t1/cl/lsst/tmp/henrique.almeida/testlephare/data'
>>> import lephare as lp
User defined LEPHAREDIR is set. Code runs depend on all required
                auxiliary data being present at /lustre/t1/cl/lsst/tmp/henrique.almeida/testlephare/data.
Default work cache at /lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/var/cache/lephare/work
                    is already linked. This is linked to the run directory:
                    /lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/var/cache/lephare/runs/20240813T173817
>>> lp.data_retrieval.get_auxiliary_data(clone=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: get_auxiliary_data() got an unexpected keyword argument 'clone'
```

Versions:
- lephare: 0.1.9 (from PyPI)
- python 3.11.9 (from conda default channel)

- [x] I have described the situation in which the bug arose, including what code was executed, information about my environment, and any applicable data others will need to reproduce the problem.
- [x] I have included available evidence of the unexpected behavior (including error messages, screenshots, and/or plots) as well as a descriprion of what I expected instead.
- [x] If I have a solution in mind, I have provided an explanation and/or pseudocode and/or task list.

**Comment by johannct** [comment]
same problem as #180 : pypi latest release is lagging.

**Comment by raphaelshirley** [comment]
I just released the latest version to pypi. I will close the issues when it has propagated and we are confident it is fixed.

**Comment by raphaelshirley** [comment]
There is an issue with the PyPI deployment. I am looking into it now. As a temporary solution you could simply remove the clone keyword.

**Comment by hdante** [comment]
Hello, everyone, tested with lephare 0.1.10 and it worked. Thanks,

**Comment by johannct** [comment]
closed as resolved with v0.1.10
